__tests__/components/community/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import "@testing-library/jest-dom";
+import CommunityCard from "@/components/cards/community";
+import { render, screen } from "@testing-library/react";
+import { community } from "../../../__mocks__/community";
+import ReduxProvider from "../../../__mocks__/provider/ReduxProvider";
+jest.mock("next/router", () => ({
+  useRouter: () => ({
+    push: jest.fn(),
+    isFallback: false,
+  }),
+}));
+
+const communityCardProps: any = {
+  showRewards: false,
+  community: community,
+};
+
+function RenderCommunityCard(props = communityCardProps) {
+  render(
+    <ReduxProvider>
+      <CommunityCard showRewards={props.showRewards} community={props.community} />
+    </ReduxProvider>
+  );
+  return screen.getByTestId("community-card");
+}
+
+describe("Community", () => {
+  it("should render the community card", () => {
+    const card = RenderCommunityCard();
+    expect(card).toBeInTheDocument();
+  });
+
+  it("should show rewards when we have rewards and show rewards when enabled", () => {
+    RenderCommunityCard({ ...communityCardProps, showRewards: true });
+    const communityCardButtonrewards = screen.getByTestId("community-rewards");
+    expect(communityCardButtonrewards).toBeInTheDocument();
```

Comment:
We are not testing much here. We should check if the values we are passing to the component are showing up as we expect

Reply:
Test cases shouldn't just cover the idea that something is rendered only. it should also check if the values and the component is working as expected if fed with the right data

Reply:
You should also be able to test if data is different how the component handles that particular scenario